email_automation/email-config.py

```diff
@@ -0,0 +1,171 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+import logging
+import sys
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+#Input the filename as an argument in command line 
+filename_raw = sys.argv[1]
+filename = filename_raw.split(".")[0]
+pool_name = "nrelopenpath-prod-" + filename
+
```

Comment:
nit: It is generally not considered good practice to write raw code like this, especially for files that are > ~ 50 lines long. You should use a `__main__ function instead.

Reply:
Added `if __name__ == "__main__":` function